wiki/Beatmapping/Circle_size/en.md

```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - CS
+---
+
+# Circle size
+
+**Circle size** (***CS***) is a difficulty setting which determines the size of [hit objects](/wiki/Hit_Objects) on [beatmaps](/wiki/Beatmaps). The value for circle size ranges from 2 to 7, where a higher value will make hit objects smaller. Most beatmaps use a value between 3 and 5, while values between 5 and 6 are sometimes used. Values between 6 and 7 are almost never used. Circle size values higher than 7 or lower than 2 can be seen when the [Hard Rock](/wiki/Game_Modifiers#hard-rock) or [Easy](/wiki/Game_Modifiers#easy) mods are used respectively.
+
+It is possible to set the circle size value outside normal bounds by manually editing the [.osu file](/wiki/osu!_File_Formats/Osu_(file_format)). However, this is considered unrankable by the [ranking criteria](/wiki/Ranking_Criteria).
+
+## osu!standard
+
+In osu!standard, only the size of hit circles and sliders are adjusted through circle size, while spinners remain unaffected. Mathematically, circle size is derived through the following formula:
+
+`r = 32 * (1 - 0.7 * (CS - 5) / 5)`, where `r` is the radius measured in osu!pixels. Alternatively, this can be written as:
```

Comment:
should just use the second version of the formula

Reply:
i dont think simplifying a formula like this and calling it a day is good when potential users may wonder why those exact values were used. I think i'll delete the simplification though.

Reply:
> when potential users may wonder why those exact values were used

the formula u kept also doesn't solve this issue either. to a reader it's way more complicated for no reason. if you want the constants to be explained further (this is already partially simplified, fwiw) pls add a todo comment instead

Reply:
poke

Reply:
will use the second version of the formula